composeml/label_times.py

```diff
@@ -19,55 +35,49 @@ def __init__(self, data=None, name=None, target_entity=None, settings=None, tran
         self.target_entity = target_entity
         self.settings = settings or {}
         self.transforms = transforms or []
+        self.plot = LabelPlots(self)
 
     @property
     def _constructor(self):
         return LabelTimes
 
     @property
-    def distribution(self):
-        labels = self.assign(count=1)
-        labels = labels.groupby(self.name)
-        distribution = labels['count'].count()
-        return distribution
+    def _is_categorical(self):
```

Comment:
I think a user should be able to provide a `label_type` parameter to override this. Label types could be "categorical" or "continuous". we can also pass this parameter through when using the label maker. for example, bin would know to pass categorical through. 

In terms of infer categorical, I'd also check for any non numeric dtyes like objects, strs, or the pandas categorical dtype

Reply:
additionally, we only want to be calculating this once when the labeltime are initialized, not every time `_is_categorical` is accessed

Reply:
What do you think about overriding it by using type hints?
```python
def labeling_function(df) -> 'categorical':
    ...
```

Reply:
Also casting label times to categorical dtype if categorical. Related to #60

Reply:
I don't think that the type hints for annotations  is very intuitive. might just be better to make it a parameter to the label maker

Reply:
okay, will make parameter for `label_type`